base/features/junit/feature.yml

```diff
@@ -1,8 +1,10 @@
-description: Adds support for the JUnit testing framework
+description: Adds support for the JUnit 5 testing framework
 dependencies:
   - scope: testCompile
-    coords: junit:junit:4.12
+    coords: org.junit.jupiter:junit-jupiter-api:5.3.1
+  - scope: testRuntime
+    coords: org.junit.jupiter:junit-jupiter-engine:5.3.1
   - scope: testCompile
-    coords: io.micronaut:micronaut-inject-java
+    coords: io.micronaut.test:micronaut-test-junit5:1.0.1
```

Comment:
Should remove the hard coded version here as it is already in the BOM https://github.com/micronaut-projects/micronaut-core/blob/master/build.gradle#L143

Reply:
I tried. It's not resolved. The reason being that, in order to test it, I had to set the profile versions to `1.0.2` and that BOM doesn't contain the version.

The underlying problem is that there is a bug preventing to create applications using a profile from Maven coordinates:

    mn create-app test-service-junit5 --profile="io.micronaut.profiles:service:1.1.0.BUILD-SNAPSHOT" --features=junit